Fix cli download (#1083)
scalableminds/webknossos-libs: webknossos/Changelog.md

- Moved functional parts of merge volume annotation CLI to Dataset and Annotation classes. [1055](https://github.com/scalableminds/webknossos-libs/pull/1055)

### Fixed

> - We've resolved an issue that was affecting the download of annotations through the Command Line Interface (CLI). Additionally, we've improved the CLI to better differentiate between dataset URLs and annotation URLs before initiating different download methods. [1083](https://github.com/scalableminds/webknossos-libs/pull/1083)

Reviewer: ```suggestion
- Fixed an issue with downloading annotations through the Command Line Interface. [#1083](https://github.com/scalableminds/webknossos-libs/pull/1083)
```

(I’d go for a more compact style in the changelog, and only mention the user-facing change, which is the fix, not the refactoring)
